apps/gallery/style/gallery.css

```diff
@@ -900,7 +900,11 @@ html[dir="rtl"] .sliderlabel {
   text-align: center;
   top: 0.9rem;
   width: 1rem;
-  -moz-margin-start: calc(-50%  + 0.1rem);
+  margin: 0 calc(-50% + 0.1rem);
+
+  /* Slider labels are numbers */
+  direction: ltr;
+  unicode-bidi: -moz-isolate;
```

Comment:
Do you really need both direction and unicode-bidi?  I would have thought that just direction:ltr would be enough.